packages/block-library/src/button/editor.scss

```diff
@@ -33,50 +33,6 @@
 	}
 }
 
-.wp-block-button__inline-link {
```

Comment:
No longer applicable since:
https://github.com/WordPress/gutenberg/commit/a74921d2b7198139a3059f650dc52d6eb5185617#diff-e335a5733e9910d51f3adb7c21b371b738fdc190e6b9b71b19ba2ad45703fea1L153-L155